subprojects/docs/src/docs/userguide/authoring-builds/incremental_builds.adoc

```diff
@@ -394,15 +400,13 @@ There are other task types where this kind of nested inputs are available:
 In the same way, this kind of modelling is available to custom tasks.
 
 [[sec:task_input_validation]]
-==== Runtime validation
+=== Task input validation
```

Comment:
:x: This isn't about `Task input validation`, this is task validation happening at runtime. And that validation validates something about task properties, e.g. whether they are annotated, if there are missing task dependencies, etc. As opposed to the static validation done by the `java-gradle-plugin`'s `validatePlugins` task.

Reply:
Do you think `Task validation at runtime` sounds better?

We should probably change the section's link as well from `task_input_validation` if it does not fit the meaning.